apps/api-frontend/src/app/app.css

```diff
@@ -1,14 +1,20 @@
-/*
- * Remove template code below
- */
-
 .app {
-  font-family: sans-serif;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
This isn't a problem per se, but you don't need to provide so many font alternatives.  Defaulting to sans-serif is probably enough